uppsrc/CtrlCore/X11Ctrl.cpp

```diff
@@ -92,12 +93,12 @@ void GuiPlatformAfterMenuPopUp()
 	Ctrl::ProcessEvents();
 }
 
-void Ctrl::PaintCaret(SystemDraw& w)
-{
-	GuiLock __;
-	if(this == caretCtrl && WndCaretVisible)
-		w.DrawRect(caretx, carety, caretcx, caretcy, InvertColor);
-}
+//void Ctrl::PaintCaret(SystemDraw& w)  //aris002
+//{
+//	GuiLock __;
+//	if(this == caretCtrl && WndCaretVisible)
+//		w.DrawRect(caretx, carety, caretcx, caretcy, InvertColor);
+//}
```

Comment:
Commented code is bad - if we do not need this functionality just remove the code completely.